notification/README.md

```diff
@@ -0,0 +1,105 @@
+# Open Data Service - Notification-Service
+
+## Getting started
+
+Make sure they keyclone container is running (see `../auth`).
```

Comment:
keycloak ;-)

Reply:
I see this typo is copied from transformation service. So lets leave it like this and fix it everywhere in a separate issue